.ci/.storybook/main.js

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+const config = require('@kbn/storybook').defaultConfig;
+const aliases = require('../../src/dev/storybook/aliases.ts').storybookAliases;
+
+config.refs = {};
+
+for(const alias of Object.keys(aliases).filter(a => a !== 'ci_composite')) {
+  // snake_case -> Title Case
+  const title = alias
+    .replace(/_/g, ' ')
+    .split(' ')
+    .map(n => n[0].toUpperCase()+n.slice(1) )
```

Comment:
FYI, I'm assuming https://github.com/elastic/kibana/pull/90447 will trigger prettier in this file, but we might also need to add `!.ci` to `.eslintignore`

Reply:
@spalger it didn't trigger on this file. I at least manually updated the license header

Reply:
Can you try adding `!.ci` to the .eslintignore file?

Reply:
Relying on manual updating is less than ideal